apps/api/src/app/shared/services/query-parser/types.ts

```diff
@@ -0,0 +1,35 @@
+export enum JsonComparisonOperatorEnum {
+  EQUAL = '==',
+  NOT_EQUAL = '!=',
+  GREATER_THAN = '>',
+  LESS_THAN = '<',
+  GREATER_THAN_OR_EQUAL = '>=',
+  LESS_THAN_OR_EQUAL = '<=',
+  IN = 'in',
+  STARTS_WITH = 'startsWith',
+  ENDS_WITH = 'endsWith',
+}
```

Comment:
Those are coming from the library we use? Assuming there are no type we can reuse between client and server?

Reply:
yes, unfortunately, there is no enum that we can reuse